src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { Accordion } from "./Accordion";
+
+export default {
```

Comment:
can we add more children so we can see how well they play?
Also, I was looking in storybook and paddings/margins look weird

Reply:
will add more children

at the moment we don't have specs for the content. Worst case is to style the content based on the needs of the single use case

Reply:
@fneves check it out

Reply:
@fneves check it out